### ISSUE TITLE
iOS 7.0 & iPhone 4s Support

### ISSUE THREAD

**Issue by LeoiOS** [matmartinez/MMNumberKeyboard#4]
On my iPhone 4s, the keyboard does not display.

**Comment by matmartinez** [comment]
Hi. The included example does indeed run on iOS 7 (and was tested on an iPhone 4s as well). Can you please provide more details about this?

**Comment by LeoiOS** [comment]
````objc
    MMNumberKeyboard *keyboard1 = [[MMNumberKeyboard alloc] initWithFrame:CGRectZero];
    keyboard1.allowsDecimalPoint = NO;
    keyboard1.delegate = self;
    
//    MMNumberKeyboard *keyboard2 = [[MMNumberKeyboard alloc] initWithFrame:CGRectZero];
//    keyboard2.allowsDecimalPoint = NO;
//    keyboard2.delegate = self;
    
    self.phoneField.inputView = keyboard1;
//    self.qqField.inputView = keyboard2;
````

like this, and then `phoneField` doesn't show any keyboard, `qqField` show system keyboard.

**Comment by HCFR** [comment]
Hi Matmartinez

Very nice work with the Keyboard :)

Got the same problem on a iPhone4 with iOS 7.1.2
Same syntax as LeoiOS

**Comment by matmartinez** [comment]
Hey there! This morning I'll have the chance to test this on iOS 7. I have an idea of what it could be. Sorry about this. Thank you both!

**Comment by matmartinez** [comment]
Thanks for your help guys! This issue is now fixed on version 0.1.4

**Comment by LeoiOS** [comment]
Thank you for fix it!

**Comment by HCFR** [comment]
Works like a charm in iOS 7.1.2 now.

Thank you Matias.

Med venlig hilsen
Lindab A/S
Hans Christian From Rebling
Webudvikler

T: +45 73 23 23 11 (dir) F: +45 73 23 23 24
M: +45 23 35 90 47 | E: hanschristianfrom.rebling@lindab.com
Lindab A/S | Finnmarken 1, Jels | DK-6630 Rødding
www.lindab.dk
Fra: Matías Martínez [mailto:notifications@github.com]
Sendt: 21. januar 2016 16:13
Til: matmartinez/MMNumberKeyboard <MMNumberKeyboard@noreply.github.com>
Cc: Rebling Hans Christian From <hanschristianfrom.rebling@lindab.com>
Emne: Re: [MMNumberKeyboard] iOS 7.0 & iPhone 4s Support (#4)


Thanks for your help guys! This issue is now fixed on version 0.1.4

**Comment by matmartinez** [comment]
Yay! Thank you all for reporting this :)
